### ISSUE TITLE
html support?

### ISSUE THREAD

**Issue by schnittstabil** [yui/yuidoc#344]
Don't know if this change was on purpose, but since #321 yuidoc autoescapes html tags.

That is because the `html` option of [markdown-it](https://github.com/markdown-it/markdown-it#init-with-presets-and-options) defaults to `false`, which can be easily fixed in a configfile:
```json
{
  "options": {
    "markdown": {
      "html": true
    }
  }
}
```

**Comment by okuryu** [comment]
Yeah, you need to turn to `true` the `html` option if you have a code which contains HTML like this.

```js
/**
 * This is the <b>MyClass</b> class.
 *
 * @class MyClass
 */
```

This might be construed as a regression, so maybe we should change to `true` this option.

**Comment by schnittstabil** [comment]
I don't know much about the old behavior, but I think if it's a regression, maybe two further options should also be taken into account:
```
linkify:      false,        // Autoconvert URL-like text to links

// Enable some language-neutral replacement + quotes beautification
typographer:  false,
```

**Comment by okuryu** [comment]
Thanks! We've used `marked` as Markdown parser previously. I've verified that turning to `true` the `html` and `linkify` option seems to be similar in the default behavior of `marked`.

But I'm not sure that we should set to `true` the `typographer` option of `markdown-it`. Do you have a confirmable example?

**Comment by schnittstabil** [comment]
As far as I can see `typographer` replaces `(c)` with `©` etc. - there are some examples at https://markdown-it.github.io. But neither I know the old behavior, nor if it makes sense to change it…

**Comment by okuryu** [comment]
IMO, `marked` doesn't have a feature same with `typographer` option, but I'd like to consider to switch to `true` the `html`, `linkify` and `typographer` options.

**Comment by okuryu** [comment]
Published `yuidocjs@0.7.0`.
